django/db/backends/mysql/base.py

```diff
@@ -302,7 +302,7 @@ def force_no_ordering(self):
         columns. If no ordering would otherwise be applied, we don't want any
         implicit sorting going on.
         """
-        return [(None, ("NULL", [], 'asc', False))]
+        return [(None, ("NULL", [], False))]
```

Comment:
Not needed for this PR, but we could likely change the logic so that mysql returns an expression instead.